app/assets/stylesheets/gallery.scss

```diff
@@ -14,6 +14,10 @@ $margin: 15px;
       top: $margin;
     }
 
+    .slide-loading {
+      background-image: image-url("blueimp-gallery/img/loading.gif");
```

Comment:
So this sets correct loading path for this image, overriding this: https://github.com/blueimp/Gallery/blob/6075850b76e511cf013ee18c238ab7ddc756a131/css/blueimp-gallery.css#L85

There must be a better way of doing this, but I could get it now